src/Components/Bookings/Bookings.js

```diff
@@ -1,10 +1,40 @@
-import React from "react";
+import React, { useState } from "react";
+import "./Bookings.css";
+import BookingsModal from "./BookingsModal/BookingsModal";
+import Backdrop from "./Backdrop/Backdrop";
 
 const Bookings = (props) => {
+	const [viewing, setViewing] = useState(false);
+
+	const onCancelHandler = () => {
```

Comment:
this is not a good function name. Please name it to reflect what it does exactly. For example `cancelBackdrop` for a function that cancels a backdrop

Reply:
done